src/api/player/server/kickPlayer.lua

```diff
@@ -0,0 +1,19 @@
+---kickPlayer
+---@param playerSrc number
+---@return void
+---@public
+function API_Player:kickPlayer(playerSrc, reason)
+    reason = (reason or _Literals.KICK_DEFAULT_MESSAGE)
+    if (playerSrc) then
+        DropPlayer(playerSrc, reason)
+    end
+    CreateThread(function()
+        while (true) do 
+            if (GetPlayerPing(playerSrc) >= 0) then
+                break
+            end
+            Wait(200)
+            DropPlayer(playerSrc, reason)
+        end
+    end)
```

Comment:
Why need this?
Especially if we dropped the player ?

Reply:
This forces the kick of the player in case the server had a latency in the execution of the function

Reply:
Oh okay. I thinked the fivem client solved this problem.